src/components/stepforms/ConditionsEditor/ConditionsGroup.vue

```diff
@@ -0,0 +1,470 @@
+<template>
+  <div
+    class="conditions-group"
+    :class="{
+      'conditions-group--with-switch': hasMultipleRows,
+    }"
+  >
+    <div v-if="hasMultipleRows" class="conditions-group__switch">
+      <div class="condition-group__switch-link" />
+      <div class="conditions-group__switch-buttons">
+        <div
+          class="conditions-group__switch-button conditions-group__switch-button--and"
+          :class="{ 'conditions-group__switch-button--active': operator === 'and' }"
+          @click="switchOperator('and')"
+        >
+          and
+        </div>
+        <div
+          class="conditions-group__switch-button conditions-group__switch-button--or"
+          :class="{ 'conditions-group__switch-button--active': operator === 'or' }"
+          @click="switchOperator('or')"
+        >
+          or
+        </div>
+      </div>
+    </div>
+    <div v-for="(condition, rowIndex) in conditions" :key="'row' + rowIndex" class="condition-row">
```

Comment:
```suggestion
    <div v-for="(condition, rowIndex) in conditions" :key="`row${rowIndex}`" class="condition-row">
```

but wouldn't `rowIndex` be enough in that case? Besides, isn't it a bit fragile to rely solely on the index?

Reply:
The underlaying structure of data is:
```
conditions: [x, y, z]
groups: [xg, yg, zg]
```

Therefore, the `v-for` need an id to identify x, y, z, xg, yg and zg.
- Adding `group` and `row` before the index distinguishes the conditions array elements from the `group` ones
- The index in the array seems enough because:
  - The order of the elements inside each array is not going to change (as operators are commutatives, there would be no need to reorder them)
  - As the content of a condition can be completly modified, and still stay the same row, I don't see any other possible identification for it.